lib/common/terminal.js

```diff
@@ -0,0 +1,17 @@
+var osascript = function(script) {
+  return 'osascript -e \'' + script + '\'';
+};
+
+var terminal = function(cmd) {
+  return 'tell application "Terminal" to do script "' + cmd + '"';
+};
+
+var runInTerminal = function (cmd) {
+  return osascript(terminal('cd {projectPath}; ' + cmd));
+};
+
+module.exports = {
+  osascript: osascript,
```

Comment:
Why do we need to expose `osascript` and `terminal`? I think `module.exports = runInTerminal;` is enough.